<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCallDialog</class>
 <widget class="QDialog" name="NewCallDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>451</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="messageTab">
    <attribute name="title">
     <string>Message</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,3">
      <item row="0" column="1">
       <widget class="QComboBox" name="messageSourceComboBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="messageMethodComboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Async:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="messageDestinationComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Destination:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Source:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Method:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="messageAsyncCheckBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QDialogButtonBox" name="messageButtonBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>341</width>
       <height>32</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Destroy</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Source</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Destination</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="destroySourceComboBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="destroyDestinationComboBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QDialogButtonBox" name="destroyButtonBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>341</width>
       <height>32</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Create</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Source</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="createSourceComboBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>New Instance</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0" columnstretch="0,0">
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Class</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="createNameLineEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="createClassesComboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QDialogButtonBox" name="createButtonBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>341</width>
       <height>32</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Add Call</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
